lib/internal/bootstrap_node.js

```diff
@@ -304,6 +306,40 @@
     };
   }
 
+  function setupProcessICUVersions() {
+    const icu = process.binding('icu');
+    if (icu) {
```

Comment:
I usually find `if (!icu) return;` a bit easier to read, but maybe that’s only because it avoids the extra indentation level, so feel free to keep this as it is.